hip-lang-config.cmake.in

```diff
@@ -65,17 +65,15 @@ find_dependency(amd_comgr)
 
 include( "${CMAKE_CURRENT_LIST_DIR}/hip-lang-targets.cmake" )
 
-# From hip-lang config directory, do three level up
-get_filename_component(_DIR "${CMAKE_CURRENT_LIST_DIR}" REALPATH)
-get_filename_component(_IMPORT_PREFIX "${_DIR}/../../../" REALPATH)
+# Find the hip-lang config file path with symlinks resolved
+# RealPath: /opt/rocm-ver/lib/cmake/hip-lang/hip-lang-config.cmake
+# Go 4 level up to get IMPORT PREFIX
+get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" REALPATH)
+get_filename_component(_IMPORT_PREFIX "${_DIR}/../../../../" ABSOLUTE)
```

Comment:
This is what I found.
```
CMAKE_CURRENT_LIST_FILE: /opt/rocm-ver/lib/cmake/hip-lang/hip-lang-config.cmake
_DIR: /opt/rocm-ver/lib/cmake/hip-lang/hip-lang-config.cmake
_IMPORT_PREFIX: /opt/rocm-ver
```
It seems giving the desired _IMPORT_PREFIX value.
_DIR is no more a directory and `"${_DIR}/../../../../"` seems very strange to me.
No clear why this is better than the old working lines.

Reply:
In the hip-runtime-amd package , the hip-lang-config.cmake is there in
/opt/rocm-ver/lib/cmake/hip-lang/hip-lang-config.cmake         # actual file
/opt/rocm-ver/hip/lib/cmake/hip-lang/hip-lang-config.cmake         #  Soft link file for backward compatibility 
The directory /opt/rocm-ver/**hip**/lib/cmake/hip-lang is not a soft link, but file is soft link
With the previous code, the _IMPORT_PREFIX may become /opt/rocm-ver/hip if run from backward compatibility path. 
Agree that _DIR is now no more a directory. Will update the variable name